tools/test_glue.py

```diff
@@ -0,0 +1,15 @@
+import sys
+from subprocess import Popen, PIPE
+
+def run_it(cmd):
```

Comment:
This is nice, but the code is not integrated in the input() function, so I suggest we move the code there and then re-submit this PR